src/components/standalone/dashboard/ThreatShieldIpCard.vue

```diff
@@ -0,0 +1,218 @@
+    serviceCounter.value = res.data.result.count
+  } catch (err: any) {
+    console.error(err)
+    error.value.title = t('error.cannot_retrieve_service_status')
+    error.value.description = t(getAxiosErrorMessage(err))
+  } finally {
+    loading.value.getServiceCounter = false
+  }
+}
+
+async function getThreatShieldConfig() {
+  try {
+    loading.value.getThreatShieldConfig = true
+    const res = await ubusCall('ns.threatshield', 'list-settings')
+    threatShieldConfig.value = res.data.data
+  } catch (err: any) {
+    console.error(err)
+    error.value.title = t('error.cannot_retrieve_threat_shield_settings')
+    error.value.description = t(getAxiosErrorMessage(err))
+  } finally {
+    loading.value.getThreatShieldConfig = false
+  }
+}
+
+function getBadgeKind(status: string) {
+  switch (status) {
+    case 'ok':
+      return 'success'
+    case 'warning':
+      return 'warning'
+    case 'disabled':
+      return 'secondary'
+    default:
+      return 'error'
+  }
+}
+
+function getBadgeText(status: string) {
+  switch (status) {
+    case 'ok':
+      return t('standalone.dashboard.active')
+    case 'warning':
+      return t('standalone.dashboard.warning')
+    case 'disabled':
+      return t('standalone.dashboard.inactive')
+    default:
+      return t('standalone.dashboard.unknown')
+  }
+}
+
+function getBadgeIcon(status: string) {
+  switch (status) {
+    case 'ok':
+      return faCheck
+    case 'warning':
+      return faWarning
+    case 'disabled':
+      return faXmark
+    default:
+      return faXmark
+  }
+}
+
+function goTo(path: string) {
+  router.push(`${getStandaloneRoutePrefix(route)}${path}`)
+}
+</script>
+
+<template>
+  <NeCard
+    :icon="['fas', 'shield']"
+    :skeletonLines="2"
+    :loading="loading.getServiceStatus || loading.getServiceCounter"
+    :errorTitle="error.title"
+    :errorDescription="error.description"
+  >
+    <!-- title slot -->
+    <template #title>
+      <NeLink @click="goTo('/security/threat-shield')">
+        {{ t('standalone.dashboard.thread_shield_ip') }}
+      </NeLink>
+    </template>
+    <div class="space-y-3">
+      <!-- status -->
+      <NeBadge
+        :kind="getBadgeKind(serviceStatus)"
+        :text="getBadgeText(serviceStatus)"
+        :icon="getBadgeIcon(serviceStatus)"
+      />
+      <!-- monitoring disabled warning -->
+      <NeTooltip v-if="isLoggingDisabled" class="inline-block">
+        <template #trigger>
+          <NeBadge
+            kind="warning"
+            :text="t('standalone.dashboard.monitoring_disabled')"
+            :icon="faWarning"
+          />
+        </template>
+        <template #content>
+          <i18n-t keypath="standalone.dashboard.monitoring_disabled_tooltip" tag="p" scope="global">
+            <template #page>
+              <NeLink invertedTheme @click="goTo('/security/threat-shield?tab=settings')">
+                {{ t('standalone.threat_shield.settings') }}
```

Comment:
I'd use a RouterLink component here (naturally wrapped by a NeLink for styling)
https://router.vuejs.org/guide/#App-vue